src/js/utils/underscore.d.ts

```diff
@@ -12,6 +12,9 @@
 
 /* eslint spaced-comment: 0 */
 /* eslint @typescript-eslint/no-empty-interface: 0 */
+/* eslint @typescript-eslint/no-shadow: 0 */
```

Comment:
We didn't even write this file. Ideally we would switch to `@types/underscore`, but I tried to do that and it causes a ton of our type definitions to fail. I will add it to our technical debt tracking internally.